atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -229,13 +229,13 @@ public String toString() {
         // this test is free/just checks a boolean and does not block; borrow is still fast
         poolConfig.setTestOnBorrow(true);
 
-        poolConfig.setMinEvictableIdleTimeMillis(TimeUnit.MILLISECONDS.convert(3, TimeUnit.MINUTES));
+        poolConfig.setMinEvictableIdleTimeMillis(TimeUnit.MILLISECONDS.convert(10, TimeUnit.MINUTES));
```

Comment:
Yeah. Do you think all of them should be (minEvictableIdleTimeMillis, timeBetweenEvictionRunsMillis, numTestsPerEvictionRun)?